_posts/2021-07-13-gs_00003_terminal_basics2.md

```diff
@@ -0,0 +1,394 @@
+means that you can only install packages that are inside the database. Probably,
+you can imagine that not all packages that exist in the world are listed there.
+Consequently, the are packages you cannot install with Pacman. You can check the
+documentation of the package if it is inside another package manager or if you
+have to install it manually. We will come back to it in a later post.
+
+# Run your software
+
+After we learned how to install software, we want to know how to run it.
+Actually, you already know how to do it. Do you remember how you run the editor
+"nano" by just typing its name into the terminal? That's it! But let's give you
+another example and have a closer look at the details.
+
+You can open a firefox browser with:
+
+    $ firefox
+
+<div class='note-normal' markdown="1">
+_Note: You also can install a browser of your choice using Pacman.
+Firefox is preinstalled with Manjaro, so we use it here._
+</div>
+
+A firefox window should open. If you want to stop firefox, you can either close
+the window or press CTRL-C inside the terminal. It will also stop if you close
+the terminal. Just have a try!
+
+In the case you stopped firefox, start it again with the command above and take
+a look at the terminal. The cursor moved to the next line, but the line is
+blank. If you try to type a command like "ls" you will note that you can type
+it, but it does not do anything. This is because you can only execute a new
+command after the previous one is completed. Here, firefox is still running. An
+easy workaround is to open a second terminal and run the "ls" command there.
+Have a try! It will work. However, there is something else you can do. Stop
+firefox with CTRL-C and run it, again, with a tiny change:
+
+    $ firefox &
+
+The "&" indicates that firefox shall run in the background. You will be prompted
+a number, which is the so-called **process id**, and you are able to enter
+further commands. If you want to stop firefox from the command line, you can
+kill it using one of the following:
+
+    $ kill <process id>
+    $ killall firefox
+
+The process id is the one you were shown when starting firefox. The "kill"
+command will, therefore, only kill that specific firefox process. The "killall"
+command will kill all firefox instances. You might wonder if you really need to
+take note of the process id, and the answer is, of course, no! Always remember
+that programmers are lazy! You can use the "ps" command to find the process id
+(PID).
+
+    $ ps
+
+There are many options you can add to the command. Check the man page for a full
+list. An often used command is
+
+    $ ps aux
+
+which gives a list of all running processes. Since that list is usually long,
+you can search for a specific process:
+
+    $ ps aux | grep firefox
+
+You should be able to find your PID and also might see that firefox created
+process children. If you kill the main process, you will kill all children as
```

Comment:
I think, this would sound better:
```suggestion
_child_ processes. If you kill the main process, you will kill all children as
```
Also, probably explain briefly what a child process is?